README.md

```diff
@@ -50,3 +50,11 @@ Full integraton tests can only be run against the current production
 deployment, as it requires TLS and DNS:
 
     ./full-tests.sh
+
+## Debug build
+
+If you `export IMAGE_NAME=proxy-debug`, just run/test will now use build of the
```

Comment:
```suggestion
If you `export IMAGE_NAME=opensafely-proxy-debug`, just run/test will now use build of the
```

Reply:
I kept the local name as just proxy-debug, as wanted to avoid any sense of it being publishable.

Reply:
fair enough